src/Storage/Commands.Storage/help/Set-AzureStorageServiceProperty.md

```diff
@@ -0,0 +1,148 @@
+---
+external help file: Microsoft.WindowsAzure.Commands.Storage.dll-Help.xml
+Module Name: Azure.Storage
+online version: 
+schema: 2.0.0
+---
+
+# Set-AzureStorageServiceProperty
+
+## SYNOPSIS
+Modifies the properties for the Azure Storage service.
+
+## SYNTAX
+
+```
+Set-AzureStorageServiceProperty [-ServiceType] <StorageServiceType> [-DefaultServiceVersion <String>]
+ [-PassThru] [-Context <IStorageContext>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Set-AzureStorageServiceProperty** cmdlet modifies the properties for the Azure Storage service.
+
+## EXAMPLES
+
+### Example 1: Set Blob Service DefaultServiceVersion to 2017-04-17
+```
+C:\PS>Set-AzureStorageServiceProperty -ServiceType Blob -DefaultServiceVersion 2017-04-17
```

Comment:
@blueww same comment about providing the corresponding output from running this command

Reply:
As we talked in the meeting, since we don't have output by default (output serviceProperties when user add -passthru, since this serviceProperties is set to server not get from server), so it's OK not add the output here.